code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -262,27 +283,21 @@
 	new /obj/item/reagent_containers/food/pill/charcoal(src)
 	new /obj/item/reagent_containers/food/pill/charcoal(src)
 
-/obj/item/storage/pill_bottle/MouseDrop(obj/over_object as obj) //Quick pillbottle fix. -Agouri

```

Comment:
I dunno why this was copy pasted over. This behavior is present for all storage based items; the only difference here is it doesn't play a little storage interaction sound when you move it from one hand to the other. 

I'm chalking this up to very bad, very old code.

Reply:
This `MouseDrop` code was literally everywhere at one point. I remember culling it when I refactored clipboards ages ago because it was useless there too.